src/sonic-yang-models/yang-models/sonic-smart-switch.yang

```diff
@@ -157,6 +160,149 @@ module sonic-smart-switch {
 			}
 		}
 		/* end of container DPU */
+
+		container REMOTE_DPU {
+			description "REMOTE_DPU part of config_db.json";
+
+			list REMOTE_DPU_LIST {
+				key "dpu_name";
+
+				leaf dpu_name {
+					description "Name of the dpu";
+					type string {
+						pattern "[a-zA-Z0-9-]+[0-9]+";
+					}
+				}
+
+				leaf type {
+					description "Type of dpu";
+					type string;
+				}
+
+				leaf pa_ipv4 {
+					description "DPU ipv4 physical address";
+					type inet:ipv4-address;
+				}
+
+				leaf pa_ipv6 {
+					description "DPU ipv6 physical address";
+					type inet:ipv6-address;
+				}
+
+				leaf npu_ipv4 {
+					description "Loopback ipv4 of remote NPU";
+					type inet:ipv4-address;
+				}
+
+				leaf npu_ipv6 {
+					description "Loopback ipv6 of remote NPU";
+					type inet:ipv6-address;
+				}
+
+				leaf dpu_id {
+					description "ID of the DPU from minigraph";
+					type string {
+						pattern [0-7];
+					}
+				}
+
+				leaf swbus_port {
+					description "TCP listening port for swbus service for this DPU. Must be 23606 + dpu_id";
+					type inet:port-number;
+					must "current() = 23606 + number(current()/../dpu_id)";
+				}
+			}
+		}
+		/* end of container REMOTE_DPU */
+
+		container VDPU {
+			description "VDPU part of config_db.json";
+
+			list VDPU_LIST {
+				key vdpu_id;
+
+				leaf vdpu_id {
```

Comment:
I think key should be guid like 833bc6fb-b8b2-47f3-93a2-63d989f0356a?

Reply:
Thanks, good catch.